app/lib/esQueryBuilder.js

```diff
@@ -5,13 +5,73 @@ function build(searchTerm) {
     body: {
       size: 30,
       query: {
-        multi_match: {
-          query: searchTerm,
-          fields: ['name^2', 'address.addressLines', 'doctors'],
-          type: 'cross_fields',
-          operator: 'and'
+        bool: {
+          should: [
+            {
+              match_phrase: {
+                name: {
+                  query: searchTerm,
+                  boost: 2,
+                  slop: 1
+                }
+              }
+            },
+            {
+              match_phrase: {
+                alternativeName: {
+                  query: searchTerm,
+                  boost: 2,
+                  slop: 1
+                }
+              }
+            },
+            {
+              common: {
+                name: {
+                  query: searchTerm,
+                  cutoff_frequency: 0.0001
+                }
+              }
+            },
+            {
+              common: {
+                alternativeName: {
+                  query: searchTerm,
+                  cutoff_frequency: 0.0001
+                }
+              }
+            },
+            {
+              nested: {
+                path: 'doctors',
+                query: {
+                  bool: {
+                    should: [
+                      {
+                        match_phrase: {
+                          'doctors.name': {
+                            query: searchTerm,
+                            boost: 2,
+                            slop: 1
+                          }
+                        }
+                      },
+                      {
+                        match: {
+                          'doctors.name': {
+                            query: searchTerm,
+                            cutoff_frequency: 0.0001
```

Comment:
The GP name search is not intended to be a common terms search and the `cutoff_frequency` parameter was left over from an earlier version of a common terms search so I've removed it.

Will be doing some more investigation of results of the surgery name common terms search using the `explain: true` query setting to see if we can be a bit more scientific on the setting of the `cutoff_frequency` value.